molgenis-navigator/src/main/frontend/src/components/Navigator.vue

```diff
@@ -17,7 +17,7 @@
                   type="button">{{ 'search-button' | i18n }}</button>
         </span>
         <span class="input-group-btn">
-          <button @click="clearQuery()" class="btn btn-secondary" :disabled="!query"
+          <button @click="reset()" class="btn btn-secondary" :disabled="!query"
```

Comment:
You did not define a reset function in your methods section, resulting in `Uncaught TypeError: t.reset is not a function`